src/Saving/settingsSave.txt

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is het de bedoeling dat dit bestand wordt getrackt door Git? Idee om dit aan de .gitignore toe te voegen misschien?